### DISCUSSION TITLE
Feature Request web player like pocketcasts or sync for multiple devices

### DISCUSSION THREAD

**Discussion by selimovd** [PaulWoitaschek/Voice#1367] [Ideas]
Hello,

first of all thank you for this awesome player!
I use it on multiple devices and also want to use it on a PC. So I always have the problem to sync the status of the played files and my bookmarks.

I think it would be awesome to at least sync the status and the bookmarks for multiple devices.
For Podcasts in my opinion the best solution is pocketcasts, as they sync the status of the played podcasts and there is also a web player, which I can use to play on any computer. This status is then also synced to the devices. Is something like that also possible for audio books?

Thanks and best regards

**Comment by zeliboba** [comment]
I also support this feature especially for bookmarksc sync. There was a feature request for bookmark export (actually many of them,  see #582 #538 #367 #121). As a workaround for these issues (and also for #548) you could implement bookmark storage in predefined location for every audiobook, for example in the same folder. In this way the bookmarks can be synced to PC and be accessible by third party applications.

**Comment by Wizek** [comment]
Seems to be a somewhat related issue: #295
